packages/types/src/types/requests/central-server/MeditrakSurveyResponseRequest.ts

```diff
@@ -59,8 +42,8 @@ export interface MeditrakSurveyResponseRequest {
    */
   timestamp?: string;
   approval_status?: string;
-  entities_created?: EntityCreated[];
-  options_created?: OptionCreated[];
+  entities_upserted?: Entity[];
+  options_created?: Omit<Option, 'id'>[];
```

Comment:
@IgorNadj I might take a look at how easy it'd be to do the auto-generated `<Model>WithoutId` types since they do crop up a lot...